app/components/primary_nav_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class PrimaryNavComponent < ViewComponent::Base
```

Comment:
That will be the third navigation component in my recent memory that we've done - first Callum's `tabs` (well, maybe not named best), then Sebastian's `sub-navigation`, and now this lovely beast.

I don't want to sound mean, but I have a feeling we are making a lot of navigation components. And I do wonder if some of them should be replaced by things like tabs (from govuk design system). Not sure if this PR is the best place to raise it, I just don't want to end up with a messy library of components stemming from rapidly changing designs :/

Reply:
Pretty sure that Seb's subnav disappeared in favour of the tabs. Our designs have two forms of navigation, so the code needs two as well.

Reply:
There was a specific interaction reason Al specified that the tabs act as links, not like in the gov design system

Reply:
Hah, so there will be PR removing all the css that Seb added? So I guess we won't need to move it to vendor?

@seb-siem-softwire I hope you are keeping track of this, and that it doesn't make you too sad!

It's probably alright to have two navigations, especially if Al thinks so. Just wanted to make sure it's not getting out of hand :)

Reply:
Seb updated Callum's tabs. They became one, in a beautiful union